content/Store Support/Shipping/About-Ascendia.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "About Ascendia"
+---
+
+Asendia is an available shipping method for the Arduino US store. You can choose this option for most orders inside and outside the United States.
+
+Beware that even though this is the cheapest shipping option, it has the longest delivery time (similar to local post delivery times). For shipments outside the U.S., the delivery can take up to one month. Also, there can be issues tracking and locating your parcel.
```

Comment:
```suggestion
Although this is a cheaper option, it has a longer delivery time (similar to local post delivery times). For shipments outside the U.S., the delivery can take up to one month. Also, there can be issues tracking and locating your parcel.
```
"Beware" may be a bit dramatic?. Changed "longest" to "longer" to avoid conflicting with [About DHL Global Mail Packet Plus Priority International](https://support.arduino.cc/hc/en-us/articles/360016495719-About-DHL-Global-Mail-Packet-Plus-Priority-International).